lib/timer.rb

```diff
@@ -0,0 +1,23 @@
+require 'time'
+
+class Timer
+    attr_reader :time 
+    attr_accessor :start, :stop
+
+    def initialize
+        @start = 0
+        @stop = 0
+    end
+
+    def start_timer 
+        @start = Time.now.to_i
+    end
+
+    def stop_timer
+        @stop = Time.now.to_i
+    end
+    
+    def elapsed_time
+        @stop - @start 
```

Comment:
To create our elapsed_time method, we figured out that Time.now.to_i returned an integer value, we were able to take that value and, using the start_timer and stop_timer methods get a difference that indicated how many seconds lapsed.